### PR TITLE
New border color

### PR DESCRIPTION
**Pull request by flopp999**
**What this PR does / why we need it**:
Change border colors on textbox

### CODE REVIEW

**Review by flopp999 — COMMENTED** [review]
comments for https://github.com/grafana/grafana/pull/22947/commits/b0a0ff1a977a31f5123ed52480f0813675d511cc

change to 2px, since other bars e.g. Dashboard-Manage-Playlist is using 2 px

brand-bottom-border is only used in:
https://github.com/grafana/grafana/blob/15b2bf3e5c73937e6ac5d2bbadf57124f4172066/public/sass/pages/_dashboard.scss#L107

brand-top-border is new and used in:
https://github.com/flopp999/grafana/blob/patch-4/public/sass/components/_infobox.scss#L5

**Comment by flopp999** [comment]
comment for https://github.com/grafana/grafana/pull/22947/commits/4f82331d5e791727edf01f1f60c5d0245d83f4d3

**Comment by torkelo** [comment]
screenshots of your changes would help :)

File: public/sass/mixins/_mixins.scss [hunk 1]
```diff
@@ -351,14 +351,14 @@
   border-top: 0;
   border-right: 0;
   border-left: 0;
-  border-bottom-width: 1px;
+  border-bottom-width: 2px;
```
Comment (flopp999) [inline public/sass/mixins/_mixins.scss]: old bar
image

new bar
image

File: public/sass/mixins/_mixins.scss [hunk 2]
```diff
@@ -351,14 +351,14 @@
...
 }
 
 @mixin brand-top-border() {
   border-image: $brand-gradient-horizontal;
   border-image-slice: 1;
   border-style: solid;
-  border-top-width: 1px;
+  border-top-width: 2px;
```
Comment (flopp999) [inline public/sass/mixins/_mixins.scss]: Explore:
old bar
image
new bar
image
Variables:
old bar
image

new bar
image

File: public/sass/components/_gf-form.scss [hunk 3]
```diff
@@ -417,7 +417,7 @@ select.gf-form-input ~ .gf-form-help-icon {
   padding: $space-lg;
   background-color: $empty-list-cta-bg;
   margin-bottom: $space-lg;
-  border-top: 3px solid $green-base;
+  @include brand-top-border();
```
Comment (flopp999) [inline public/sass/components/_gf-form.scss]: New API keys
old
image

new
image

**Comment by mckn** [comment]
Great work @flopp999! Would be awesome to hear your ideas behind the changes since it only has to do with look and feel :) I'm just curious.

**Comment by flopp999** [comment]
> Great work @flopp999! Would be awesome to hear your ideas behind the changes since it has to do with look and feel :) I'm just curious.

Thanks. I just saw one border that was green and then I got the idea to check everywhere and found some places that was not using the Grafana-color(yellow/red).
I want it to be the same pattern everywhere and to be consistent, so the user recognize when they clicking around the menus

**Comment by tskarhed** [comment]
It's really cool of you to just dig in and do some work! I want to see more people do that!

In this case I would say that the colors are the way they are for a reason. Blue communicates that it is a note to the user - green communicates a positive action (something being added).

**Comment by flopp999** [comment]
> It's really cool of you to just dig in and do some work! I want to see more people do that!
> 
> In this case I would say that the colors are the way they are for a reason. Blue communicates that it is a note to the user - green communicates a positive action (something being added).

Thank you, if I can help you do some work I will try to do it :)

**Comment by tskarhed** [comment]
> Thank you, if I can help you do some work I will try to do it :)

@flopp999 You can always have a look at issues with the `help wanted` label :)
https://github.com/grafana/grafana/labels/help%20wanted

**Comment by mckn** [comment]
> > It's really cool of you to just dig in and do some work! I want to see more people do that!
> > In this case I would say that the colors are the way they are for a reason. Blue communicates that it is a note to the user - green communicates a positive action (something being added).
> 
> Thank you, if I can help you do some work I will try to do it :)

Great feedback by @tskarhed and I agree with him. There are lots of things that we would love to have your help with.

I will close this PR since it won't be merged due to the reasons @tskarhed pointed out. Would that be okay with you @flopp999?

**Comment by flopp999** [comment]
> I will close this PR since it won't be merged due to the reasons @tskarhed pointed out. Would that be okay with you @flopp999?

Absolutely, no problem. I closed it myself :)

**Pull request closed without merging**